pandasai/llm/base.py

```diff
@@ -107,3 +113,89 @@ def generate_code(self, instruction: str, prompt: str) -> str:
             str: Code
         """
         return self._extract_code(self.call(instruction, prompt, suffix="\n\nCode:\n"))
+
+
+class BaseOpenAI(LLM, ABC):
```

Comment:
I'd say let's either create a `base_openai.py` or move also the `base_hf.py` here, so that all the "base" models are grouped. What do you think?

Reply:
I'd prefer having everything in `base.py`, but it's quite subjective. Since they all are base classes for the "backend" of pandasAI and they are small classes, I believe a unique file would serve the purpose. However the other way around would be fine as well in my opinion so you tell me :)

Reply:
Do you want me to move the content of `test_base_hf.py` inside `test_base.py` as well ?

Reply:
Would be great! @mspronesti